content/blog/posts/texas-attorney-general’s-sites-for-death-and-shooting-reports-go-dark.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Texas Attorney General’s Sites for Death and Shooting Reports Go Dark "
+subtitle: ""
+date: 2021-03-20
+authors:
+  - Eva Ruth Moravec
+---
+For five weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) have been malfunctioning, restricting access to records that detail important interactions between civilians and law enforcement. 
+
+Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that give people a glimpse at incidents like an officer’s use of deadly force, an incarcerated individual’s death and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except that part of the process is currently broken. 
+
+On Feb. 15, I noticed that both repositories were empty. After a couple of weeks, some reports appeared, but both collections are still missing all reports filed in 2021 and nearly all filed in 2020, too. 
+
+State laws require agencies to file these reports with the Texas Attorney General within 30 days of an incident in which an [officer is shot or shoots someone](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.2.htm#2.139), or when [someone dies in their custody](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.49.htm). The office then posts these reports online, which – for the officer-involved shooting reports – is a move stipulated in statute. 
+
+Said another way: By not publishing the reports on officer-involved shootings, the OAG is violating state law. 
+
+Back when the sites went down, I sent an email informing the office as such, as I have done in the past. On Feb. 22, I heard back:
+
+> Hi Ms. Moravec,
+>
+> The OAG was closed all last week due to the weather and the electrical outages.  I have reported the situation to IT this morning and they let me know they are working on it.
+>
+> As soon as I hear back from them, I will let you know.  Thank you for reporting this to us.
+
+IT then followed up that “Salesforce recently rolled out another update,” which caused the outage. About a week later, many of the cases had been restored, but hundreds from this year and last year remain missing, which I reported to the Attorney General’s office. 
+
+This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to his death isn't immediately visible online. Reports (once they are filed) and data are still obtainable through open records requests. 
```

Comment:
```suggestion
This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to the death isn't visible online. That being said, reports (once they are filed) and data are still obtainable through open records requests. 
```

Reply:
I think it is important to take out the "immediately" since the turn around time isn't that fast usually as they milk the 30 day allowance